File: programers_Lv0/문자 개수 세기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181902?language=javascript
const alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";

function solution(my_string) {
  var answer = [];

  [...alpha].forEach((el) => {
    answer.push(
      [...my_string].reduce((acc, curr) => {
        acc += el === curr ? 1 : 0;
        return acc;
      }, 0)
    );
  });

  return answer;
}

console.log(solution("Programmers"));

File: programers_Lv0/문자 개수 세기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181902?language=python3
import string

alpha = string.ascii_uppercase + string.ascii_lowercase

def solution(my_string):
    answer = []

    for i in alpha:
        answer.append(my_string.count(i))

    return answer

print(solution("Programmers"))
